### DISCUSSION TITLE
Your account has been flagged

### DISCUSSION THREAD

**Discussion by shiran-sandaruwan-9x** [community/community#53542] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I don’t understand what happened to my account.
Would you tell me what to do now?
![Screenshot (2)](https://user-images.githubusercontent.com/73925287/233844369-92c55a0a-797d-48af-88a4-6751b98f42cb.png)

**Comment by cehfisher** [comment]
Hi @shiran-sandaruwan-9x - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!

**Comment by queenofcorgis** [comment, marked as answer]
Hi @shiran-sandaruwan-9x ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
